src/2_RTB/PianoDiProgetto/sections/AnalisiDeiRischi.typ

```diff
@@ -32,6 +32,17 @@ Il team potrebbe incontrare sfide nell'apprendere l'utilizzo e nell'integrazione
     - Norme di Progetto: tutti i dettagli del _way of working_ vengono concordati durante i meeting interni e normati all'interno delle Norme di Progetto. In caso di dubbi o incertezze, il riferimento a queste norme o la richiesta di chiarimenti all'Amministratore del progetto rappresentano un'importante risorsa per la risoluzione tempestiva di problemi legati agli strumenti e alle procedure;
     - Inclusione di dubbi nell'Ordine del giorno: l'inclusione dei dubbi e delle incertezze nelle proposte dell'ordine del giorno del meeting interno successivo consente al team di affrontare in modo strutturato e tempestivo le difficoltà riscontrate, garantendo un ambiente in cui tali problemi possano essere risolti in modo collaborativo e efficiente.
 
+=== *RT3 - Produzione di codice poco comprensibile*
+Data l'inesperienza dei membri che lo compongono, il team potrebbe produrre codice difficile da comprendere, contorto. Questo unito ad un'eventuale carenza di relativa documentazione potrebbe influenzare negaitivamente i tempi di sviluppo. 
+  - *Identificazione*: un segnale evidente della manifestazione di questo rischio è la crescita delle discussioni relative all'interpretazione del codice tra i componenti del team.
+  - *Valutazione*:  Pericolosità: media; Occorrenza: media.
+  - *Prevenzione*: 
+    - Code Reviews: effettuare revisioni regolari del codice; 
+    - Continuo Refactoring: integrando all'interno del processo di sviluppo frequenti attività di Refractoring si migliora notevolmente la comprensibilità del codice; 
+    - Documentazione Adeguata: incoraggiare la produzione di una documentazione approfondita per il codice; 
+    - _Convenzioni di codifica_:  seguire le stesse linee guida porta il team a standardizzare il modo in cui il codice viene scritto, di conseguenza ne aumenta la chiarezza e la mantenibilità. 
```

Comment:
perche' e' in corsivo? dovrebbe essere termine di glossario?

Reply:
sincero non penso, tolgo il corsivo